src/Config/DatabasePrimitives/DatabaseObject.cs

```diff
@@ -98,6 +101,8 @@ public class DatabaseStoredProcedure : DatabaseObject
 {
     public DatabaseStoredProcedure(string schemaName, string tableName)
         : base(schemaName, tableName) { }
+
+    public DatabaseStoredProcedure() { }
```

Comment:
What case is it required to add these constructors?

Reply:
I need during deserialization to create the objects accordingly. without this my deserialization fails